application/ledger_query_service/src/bin/ledger_query_service.rs

```diff
@@ -135,13 +132,13 @@ fn create_ethereum_routes(
     let web3_client = Arc::new(Web3::new(transport));
 
     {
-        let transaction_query_result_repository = transaction_query_result_repository.clone();
-        let block_query_result_repository = block_query_result_repository.clone();
-        let log_query_result_repository = log_query_result_repository.clone();
-
         let block_query_repository = block_query_repository.clone();
-        let log_query_repository = log_query_repository.clone();
         let transaction_query_repository = transaction_query_repository.clone();
+        let log_query_repository = log_query_repository.clone();
+
+        let block_query_result_repository = block_query_result_repository.clone();
+        let transaction_query_result_repository = transaction_query_result_repository.clone();
+        let log_query_result_repository = log_query_result_repository.clone();
```

Comment:
❓Why randomly resorting them?

Reply:
These two blocks were sorted differently before the change, now they're sorted in the same order. Only did it for consistency.

Reply:
Also the query repositories should naturally go before the query result repositories :laughing: